cluster/manifests/skipper/deployment.yaml

```diff
@@ -3,16 +3,25 @@
 {{ $main_image := print "container-registry.zalando.net/teapot/skipper-internal:" "v0.21.211-1033" }}
 {{ $canary_image := "container-registry.zalando.net/teapot/skipper-internal:v0.21.212-1034" }}
 
-
 {{/* Optional canary arguments separated by "[cf724afc]" to allow whitespaces, e.g. "-foo=has a whitespace[cf724afc]-baz=qux" */}}
 {{ $canary_args := "" }}
 
+{{ $cluster_internal_cidrs := stringSlice "10.2.0.0/15" .Values.vpc_ipv4_cidr }}
```

Comment:
We can also make it local inside `{{ define "skipper-ingress" }}` template and then we also don't need to pass it as template argument.

Reply:
I'm not totally sure how you would like to structure it. You're welcome to push a commit to change it as you suggest.